ruby/private.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Add note to readme that one should not use these keys for his app. He should generate his own pair.